addon/webextension/selector/shooter.js

```diff
@@ -221,6 +221,15 @@ this.shooter = (function() { // eslint-disable-line no-unused-vars
   };
 
   exports.sendEvent = function(...args) {
+    let maybeOptions = args[args.length - 1];
+
+    if (typeof maybeOptions === "object") {
+      maybeOptions.incognito = browser.extension.inIncognitoContext;
+      args[args.length - 1] = maybeOptions;
```

Comment:
This line isn't necessary – you're modifying the object in-place, so it'll stay in the arguments in its modified form.